projects/collection-editor-library/src/lib/components/editor-base/editor-base.component.ts

```diff
@@ -138,7 +138,7 @@ export class EditorBaseComponent implements OnInit, OnDestroy, AfterViewInit {
         break;
       case 'onFormStatusChange':
         const selectedNode = this.treeService.getActiveNode();
-        if (selectedNode.data.root) { this.submitFormStatus = event.event.isValid; }
+        if (selectedNode && selectedNode.data.root) { this.submitFormStatus = event.event.isValid; }
```

Comment:
`selectedNode.data` will always be there?

Reply:
Yes